src/components/users/views.tsx

```diff
@@ -101,3 +126,195 @@ export function UserPage(props: IUserPageProps): ReactElement {
+  return <div className="govuk-grid-row">
+    <div className="govuk-grid-column-two-thirds">
+      <div className="govuk-panel govuk-panel--confirmation">
+        <h1 className="govuk-panel__title">{props.title}</h1>
+        <div className="govuk-panel__body">{props.message}</div>
+      </div>
+
+      <p className="govuk-body">
+        {props.footnote}
+      </p>
+    </div>
+  </div>;
+}
+
+export function PasswordResetSetPasswordForm(props: IPasswordResetSetPasswordFormProperties): ReactElement {
+  return <div className="govuk-grid-row">
+    <div className="govuk-grid-column-two-thirds">
+      {props.passwordMismatch || props.passwordDoesNotMeetPolicy ? <div
+        className="govuk-error-summary"
+        aria-labelledby="error-summary-title"
+        role="alert"
+        tabIndex={-1}
+        data-module="govuk-error-summary"
+      >
+        <h2 className="govuk-error-summary__title" id="error-summary-title">
+          There is a problem
+        </h2>
+
+        <div className="govuk-error-summary__body">
+          <ul className="govuk-list govuk-error-summary__list">
+            {props.passwordMismatch
+              ?  <li><a href="#password">You need to type in the same password twice</a></li>
+              : <></>
+            }
+            {props.passwordDoesNotMeetPolicy
+              ?  <li><a href="#password">Your password should meet our password policy</a></li>
+              : <></>
+            }
+          </ul>
+        </div>
+      </div> : <></>}
+
+      <h1 className="govuk-heading-l">Password reset</h1>
+      <p className="govuk-body">You should set a secure password which:</p>
+      <ul className="govuk-list govuk-list--bullet">
+        <li>is at least 12 characters long</li>
+        <li>includes both an uppercase and lowercase character</li>
+        <li>includes a number</li>
+        <li>
+          includes one of <code>-_+:;&lt;&gt;[]()#@£$%^&amp;!</code>
+        </li>
+        <li>you do not use anywhere else (eg for another website)</li>
+      </ul>
+
+      <form method="post" className="govuk-!-mt-r6">
+        <input type="hidden" name="_csrf" value={props.csrf} />
+
+        <div className={`govuk-form-group ${props.passwordMismatch ? 'govuk-form-group--error' : ''}`}>
+          <input type="hidden" name="code" value={props.code} />
+
+          <label className="govuk-label" htmlFor="password">
+            New Password
+          </label>
+
+          {props.passwordMismatch ? <span id="password-error" className="govuk-error-message">
+            <span className="govuk-visually-hidden">Error:</span>{' '}
+            You need to type in the same password twice
+            </span> : <></>}
+
+          {props.passwordDoesNotMeetPolicy ? <span id="password-error" className="govuk-error-message">
+            <span className="govuk-visually-hidden">Error:</span>{' '}
+            {props.passwordDoesNotMeetPolicyMessage!}
+          </span> : <></>}
+
```

Comment:
each input should be in their own `govuk-form-group`, currently they're both in the same one, hence no spacing
and error message styling (as `form-group--error` class needs to be added to the div.

when there's an error on the input it should have the below styling
<img width="430" alt="Screenshot 2020-06-12 at 12 55 35" src="https://user-images.githubusercontent.com/3758555/84500236-09fab880-acac-11ea-8fff-4180feeaffaf.png">